src/mpl/src/gpu/mpl_gpu_ze.c

```diff
@@ -23,15 +23,20 @@ MPL_SUPPRESS_OSX_HAS_NO_SYMBOLS_WARNING;
 #include "uthash.h"
 
 static int gpu_initialized = 0;
-static int device_count;
-static int max_dev_id;
+static uint32_t device_count;
+static uint32_t max_dev_id;
 static char **device_list = NULL;
 #define MAX_GPU_STR_LEN 256
 static char affinity_env[MAX_GPU_STR_LEN] = { 0 };
 
-static int *local_to_global_map;        /* [device_count] */
+static int *local_to_global_map;        /* [global_ze_device_count] */
 static int *global_to_local_map;        /* [max_dev_id + 1]   */
```

Comment:
> Aren't both array of the same size? The different comments are misleading.

They can be different sizes. `local_to_global_map` will be of size == number of visible local devices; `global_to_local_map` will be of size == number of visible devices by all node-local processes

> Commit "mpl/ze: update prefix on variables to avoid confusion" - please add commit message summarizing the changes and rationale.

:+1: